cli/kubernetes/provider.go

```diff
@@ -635,3 +635,7 @@ func loadKubernetesConfiguration(kubeConfigPath string) clientcmd.ClientConfig {
 		},
 	)
 }
+
+func IsPodRunning(pod *core.Pod) bool {
```

Comment:
why starts with capital letter? is it exposed method?

Reply:
fixed